docs/datacard.py

```diff
@@ -0,0 +1,609 @@
+                        labels = [str(label) for label in labels]
+                        # Avoid `_` at end of label,
+                        # as this has special meaning in RST (link)
+                        labels = [
+                            label[:-1] + r'\_'
+                            if label.endswith('_')
+                            else label
+                            for label in labels
+                        ]
+                        labels = limit_presented_samples(
+                            labels,
+                            15,
+                            replacement_text='[...]',
+                        )
+                    fp.write(f', "{", ".join(labels)}"')
+                if has_mappings:
+                    if not isinstance(scheme.labels, (str, dict)):
+                        mappings = ''
+                    else:
+                        labels = scheme._labels_to_dict()
+                        # Mappings can contain a single mapping
+                        # or a deeper nestings.
+                        # In the first case we just present ✓,
+                        # in the second case the keys of the nested dict.
+                        # {'f': 'female', 'm': 'male'}
+                        # or
+                        # {'s1': {'gender': 'male', 'age': 21}}
+                        mappings = list(labels.values())
+                        if isinstance(mappings[0], dict):
+                            # e.g. {'s1': {'gender': 'male', 'age': 21}}
+                            mappings = sorted(list(mappings[0].keys()))
+                            mappings = f'{", ".join(mappings)}'
+                        else:
+                            # e.g. {'f': 'female', 'm': 'male'}
+                            mappings = '✓'
+                        fp.write(f', "{mappings}"')
+                fp.write('\n')
+
+
+def create_datasets_page(datasets: typing.Sequence):
+    r"""Create overview page of datasets."""
+    # Create CSV file with overview of datasets
+    tuples = [
+        (
+            dataset.name_link,
+            dataset.short_description,
+            dataset.license_link,
+            dataset.version,
+            dataset.schemes,
+        )
+        for dataset in datasets
+    ]
+    df = pd.DataFrame.from_records(
+        tuples,
+        columns=['name', 'description', 'license', 'version', 'schemes'],
+        index='name',
+    )
+    csv_file = 'datasets.csv'
+    df.to_csv(csv_file)
+    # Create RST file showing CSV file
+    # and adding links to all data cards
+    rst_file = 'datasets.rst'
+    with open(rst_file, 'w') as fp:
+
+        fp.write('.. _datasets:\n')
+        fp.write('\n')
+        fp.write('Datasets\n')
+        fp.write('========\n')
+        fp.write('\n')
+        fp.write('Available datasets in audb_ as of |today|.\n')
+        fp.write('Only the latest version per dataset is shown.\n')
```

Comment:
```suggestion
        fp.write('For each dataset, the latest version is shown.\n')
```

Reply:
I wonder if it makes sense we also provide a list of older versions somewhere in the data card?

Reply:
I'm not completely sure about this.
The problem is that we don't provide any information for the other versions.
E.g. you could expand the table at the top of the data card by adding a line "other versions" and list all other versions (or maybe restrict to ~20 and use '...' in between) there. But I would then expect that I can click the other version and see the corresponding data card.

We could support this as well by storing the cards not under `datasets/emodb.html`, but `datasets/emodb-1.3.0.html` and adding cards for all other versions, but I'm a little bit afraid that this will result in very long build times with not that much of a benefit.

Reply:
I agree that providing information for all previous versions might be out of scope. So let's stick to the latest version then.